docs/faq.rst

```diff
@@ -130,3 +130,28 @@ How do I shutdown a KSQL environment?
    .. code:: bash
 
        $ confluent destroy
+
+========================================================
+Can KQL connect to a Apache Kafka cluster over SASL_SSL?
+========================================================
+
+Yes! Internally, KSQL uses standard Kafka Consumers and Producers, which you
```

Comment:
suggest: `Yes.` (no exclamation point). it kind sounds like you're yelling at the user :)

Reply:
done